docs/user-guide/src/developer_environment/tryit.md

```diff
@@ -39,7 +38,7 @@ For execution with VMs
 - Setup passwordless sudo access
 
 ```console
-  $ sudo visudo
+  sudo visudo
```

Comment:
This is not correct IMO. If the code block is `console` then there should be a prompt. If it is `bash` or `shell` then no prompt.

Reply:
The linter rule is that you can only show prompt if you show output too.
Arguably the original `console` is just wrong, it should be `shell`.

Reply:
Makes sense! Can you change it to shell?

Reply:
Done